### ISSUE TITLE
Is _logits_to_prob_fn step operating on every query or a batch of queries?

### ISSUE THREAD

**Issue by AaamberW** [AliVard/trust-bias-CIKM2020#3]
During updating trust bias param stage, in specific this [line](https://github.com/AliVard/trust-bias-CIKM2020/blob/5b8031b2eb47b28bfecb26e4c7b2eed04d8d6ed5/trust_bias_EM/RelevanceProbs.py#L201) rel_probs = np_softmax_prob(logits), looks like it's applied to a batch of queries without considering query_id. Also the paper(section 5, near formula 41) seems to suggest this logits to probability transformation is intended for individual queries. 

Could you help me to reconcile this?
